.travis.yml

```diff
@@ -0,0 +1,52 @@
+sudo: false
+language: python
+python: 2.7
+
+# This is a big repository so break Travis CI work into 4 CI jobs.
+env:
+  - CHUNK=0
+  - CHUNK=1
+  - CHUNK=2
+  - CHUNK=3
+
+before_install:
+  - export GALAXY_REPO=https://github.com/ErasmusMC-Bioinformatics/galaxytools-emc
+  - export GALAXY_RELEASE=release_16.07
+  - export PLANEMO_CONDA_PREFIX="$HOME/conda"
+
+install:
+  - pip install flake8 planemo
+  - planemo conda_init
+  - export PATH="$PLANEMO_CONDA_PREFIX/bin:$PATH"
+  - conda create -y -q -c bioconda --name iuc_conda samtools=0.1.19 bcftools
+  - . activate iuc_conda
+  - planemo --version
+  - conda --version
+  - git diff --quiet "$TRAVIS_COMMIT_RANGE" -- ; GIT_DIFF_EXIT_CODE=$?
+  - |
+    if [ "$GIT_DIFF_EXIT_CODE" -gt 1 ] ; then
+      git remote set-branches --add origin master;
+      git fetch;
+      TRAVIS_COMMIT_RANGE=origin/master...;
+    fi
+  - echo $TRAVIS_COMMIT_RANGE
+  - |
+      planemo ci_find_repos --exclude_from .tt_blacklist \
```

Comment:
yep, I would say we do our best to keep this file empty, would just argue to stay as in-sync with IUC as possible, if we keep an empty file with this name we don't have to change the travis.yml and can easily incorporate future changes from IUC (this goes for all these comments here, they are good points but we should decide best practices in IUC and just copy those here imho)